### PR TITLE
feat: add authentication component

### PR DESCRIPTION
**Pull request by Sahil25061999**
Added authentication pages.
structured the pages into its own directory

### CODE REVIEW

**Review by manish-GH — APPROVED** [review]

File: pages/authentication/authentication.css [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  font-weight: 600;
+  margin: 14px 0;
+}
+
+.line {
+  display: inline-block;
+  width: 45%;
+  height: 1.5px;
+  margin: 3px;
+  background-color: rgb(136, 136, 136);
```
Comment (manish-GH) [inline pages/authentication/authentication.css:103]: use a variable here

**Review by manish-GH — APPROVED** [review]
You could add a cursor pointer on hover for the checkbox input

Everything else looks good. The auth pages can be merged.

Your landing page is awesome, carousel and everything. Nice work

File: pages/authentication/authentication.css [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+}
+
+.input-container {
+  position: relative;
+  margin-bottom: 1rem;
+}
+
+.input-container label {
+  font-size: 14px;
+  margin-bottom: 5px;
```
Comment (Ankur9669) [inline pages/authentication/authentication.css:48]: fix: use rems for font size

File: pages/authentication/authentication.css [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+.remember-me-container {
+  display: flex;
+  flex-direction: row;
+  align-items: baseline;
+  justify-content: space-between;
+}
+
+.remember-me-container label {
+  font-size: 14px;
+  font-weight: 400;
```
Comment (Ankur9669) [inline pages/authentication/authentication.css:76]: Same

**Review by Ankur9669 — APPROVED** [review]
Code looks good to merge

**Pull request merged**